src/server/webpack.config.js

```diff
@@ -71,12 +71,16 @@ if (fs.existsSync(customConfigPath)) {
     logger.info('=> Loading custom webpack loaders.');
     config.module.loaders =
       config.module.loaders.concat(customConfig.module.loaders);
+    delete customConfig.module.loaders;
   }
 
   if (customConfig.plugins) {
     logger.info(' => Loading custom webpack plugins.');
     config.plugins = config.plugins.concat(customConfig.plugins);
+    delete customConfig.plugins;
   }
+
+  config = { ...config, customConfig };
```

Comment:
Actually, this is not what we should do. It's not possible to extend like this since it'll replace our configurations. But I mention is we can import his main webpack config into the webpack config in `.storybook` directory and get any modules he use.